src/views/OrderList.vue

```diff
@@ -114,16 +134,16 @@ const isSaveButtonDisabled = computed(() => {
 
     <!-- Modal -->
     <dialog id="order-detail" :open="modalIsOpen" v-if="modalIsOpen">
-    <article class="update-order-modal">
+    <article class="update-order-modal" v-if="editBtnEnabled">
     <header>
         <a href="#close"
         aria-label="Close"
         class="close"
         data-target="#order-detail"
         @click="closeModal">
         </a>
-        <h5>Update Order</h5>
-        <h6>#SLN: {{ currentOrder?.sln }}</h6>
+        <h5 class="modal-title">Update Order</h5>
+        <h6>#SLN: {{ currentOrder?.sln }}<span class="edit-icon"><i class="fa-solid fa-pen-to-square fa-xl" style="color: #c0ca33;"  @click.prevent="editBtnEnabled = !editBtnEnabled" ></i></span></h6>
```

Comment:
Place the icon inside an `<a href="#">` anchor tag and use `@click.prevent` on the anchor tag instead.
The icon itself will remain inside the anchor tag.


```suggestion
                <h6>#SLN: {{ currentOrder?.sln }}<a href="#" @click.prevent="editBtnEnabled = !editBtnEnabled" class="edit-icon"><i class="fa-solid fa-pen-to-square fa-xl" style="color: #c0ca33;" ></i></a></h6>
```

Also, remove inline styling and create a CSS class.

Reply:
done!